examples/integrations/micronaut/data/pom.xml

```diff
@@ -31,6 +31,9 @@
     <version>1.0.0-SNAPSHOT</version>
     <name>Helidon Examples Integration Micronaut Data</name>
 
+    <properties>
+        <version.lib.micronaut.validations>4.2.0</version.lib.micronaut.validations>
```

Comment:
This workaround should not be necessary. I think the problem comes from Helidon snapshot not being released and therefore, this job has access to the pom dependencies before your changes in Helidon core.
We should:
- Revert your changes without hard coding the version in the pom
- Release the snapshot in Helidon core with your changes to Micronaut versions, GAV etc
- Once done, trigger a new build on your PR that will now grab the correct dependency pom. (This does not requires a empty commit to trigger a new build, it can be done manually)

Reply:
The examples pipeline performs a build of the Helidon core workspace (prime-build) before building the examples. So a snapshot release of Helidon core should not be needed.